Kudu.Core/Jobs/ContinuousJobsManager.cs

```diff
@@ -57,6 +62,39 @@ public void EnableJob(string jobName)
             continuousJobRunner.EnableJob();
         }
 
+        public async Task<HttpResponseMessage> RequestPassthrough(string jobName, string path, HttpRequestMessage request)
+        {
+            using (var client = new HttpClient())
+            {
+                var forwardRequest = GetWebJobForwardRequest(jobName, path, request);
+                return await client.SendAsync(forwardRequest);
+            }
+        }
+
+        private HttpRequestMessage GetWebJobForwardRequest(string jobName, string path, HttpRequestMessage original)
+        {
+            var jobRunner = GetJobRunner(jobName);
+            var webJobBaseUri = new Uri(string.Concat("http://", Localhost, ":", jobRunner.WebJobPort));
+            var fullUriPath = new Uri(webJobBaseUri, path);
+            var clone = new HttpRequestMessage(original.Method, fullUriPath);
+
+            if (original.Method != HttpMethod.Get)
+            {
+                clone.Content = original.Content;
+            }
+
+            original.Headers.Aggregate(clone.Headers, (a, b) =>
```

Comment:
Please add unit tests for this request clone logic to verify it clones various request types properly, headers, etc.